fastplotlib/graphics/line_collection.py

```diff
@@ -30,6 +29,69 @@ def __init__(
             *args,
             **kwargs
     ):
+        """
+        Create a Line Collection
+
+        Parameters
+        ----------
+
+        data: list of array-like or array
+            List of line data to plot, each element must be a 1D, 2D, or 3D numpy array
+            if elements are 2D, interpreted as [y_vals, n_lines]
+
+        z_position: list of float or float, optional
+            | if ``float``, single position will be used for all lines
+            | if ``list`` of ``float``, each value will apply to the individual lines
+
+        thickness: float or list of float, default 2.0
+            | if ``float``, single thickness will be used for all lines
+            | if ``list`` of ``float``, each value will apply to the individual lines
+
+        colors: str, RGBA array, list of RGBA array, or list of str, default "w"
+            | if single ``str`` such as "w", "r", "b", etc, represents a single color for all lines
+            | if single ``RGBA array`` (tuple or list of size 4), represents a single color for all lines
+            | if ``list`` of ``str``, represents color for each individual line, example ["w", "b", "r",...]
+            | if ``RGBA array`` of shape [data_size, 4], represents a single RGBA array for each line
+
+        cmap: list of str or str, optional
+            | if ``str``, single cmap will be used for all lines
+            | if ``list`` of ``str``, each cmap will apply to the individual lines
+            **Note:** ``cmap`` overrides any arguments passed to ``colors``
+
+        name: str, optional
+            name of the line collection
+
+        args
+            passed to GraphicCollection
+
+        kwargs
+            passed to GraphicCollection
+
+        Examples
+        --------
+        .. code-block:: python
+
+            from fastplotlib import Plot
+            from fastplotlib.graphics import LineCollection
+            # creating data for sine and cosine waves
+            xs = np.linspace(-10, 10, 100)
+            ys = np.sin(xs)
+            sine = np.dstack([xs, ys])[0]
+            ys = np.cos(xs) + 5
+            cosine = np.dstack([xs, ys])[0]
+            # creating plot
+            plot = Plot()
+            # creating a line collection using the sine and cosine wave data
+            line_collection = LineCollection(data=[sine, cosine], cmap=["Oranges", "Blues"], thickness=20.0)
+            # add graphic to plot
+            plot.add_graphic(line_collection)
+            # show plot
+            plot.show()
+            # change the color of the sine wave to white
+            line_collection[0].colors = "w"
+            # change certain color indexes of the cosine data to red
+            line_collection[1].colors[0:15] = "r"
```

Comment:
show another example that you can do slicing `line_collection[1:].colors[35:70] = "b"`